content/release/ReleaseNotes370.md

```diff
@@ -0,0 +1,37 @@
+<!--
+title: "Contrast 3.7.0 - January 2020"
+description: "Contrast 3.7.0 January 2020"
+tags: "3.7.0 January Release Notes"
+-->
+
+## Bug Fixes
+
+* Ruby token authentication timed out when running Contrast with Passenger 6.0.
+
+* Some customers were not able to use RBAV with Python
+
+* For the Python agent, handle_exception was not raising the original exception. Now it does with the entire stack trace.
+
+* Node Agent 2.10.1 was crashing with Express based applications.
+
+## Agent Updates
+
+## New and Improved Features
+
+PDF Compliance Reports now include the latest Payment Card Industry (PCI) Security Standard version: PCI 3.2.1
+
+### Java Agent Summary
+
+Accuracy improved for Assess and Protect, in particular, data flow accuracy for Java 11 applications. Fixed Assess vulnerability reporting for Mulesoft based applications. Added Route Based Auto Verification (RBAV) support and improved configuration error reporting.
```

Comment:
Consider moving the following to `Bug Fixes` section:

> Fixed Assess vulnerability reporting for Mulesoft based applications.

Reply:
Thank you, moved.